src/bin/help.ts

```diff
@@ -0,0 +1,230 @@
+import chalk from "chalk";
+
+import { logLine } from "../shared/cli/lines.js";
+
+export function logHelpText(version: string): void {
+	logLine();
+
+	logLine(chalk.cyan(`✨ template-typescript-node-package v${version} ✨`));
+
+	logLine(
+		chalk.cyan(
+			`
+      A quickstart-friendly TypeScript template with comprehensive formatting, 
+      linting, releases, testing, and other great tooling built-in.
+      `,
+		),
+	);
+
+	logLine();
+
+	logLine(chalk.black.bgGreenBright("Core options:"));
+
+	logLine(
+		chalk.cyan(
+			`
+      --base: Whether to scaffold the repository with:
+        • everything: that comes with the template (${chalk.cyanBright.bold(
+					"recommended",
+				)})
+        • minimum: amounts of tooling, essentially opting out of everything
+        • prompt: for which portions to exclude
+      `,
+			`
+      --create-repository: Whether to create a corresponding repository on 
+        github.com (if it doesn't yet exist)
+      `,
+			`
+      --mode: Whether to:
+        • create: a new repository in a child directory
+        • initialize: a freshly repository in the current directory
+        • migrate: an existing repository in the current directory
+      `,
+			`
+      --description (${chalk.cyanBright.bold(
+				"string",
+			)}): Sentence case description of the repository 
+        (e.g. A quickstart-friendly TypeScript package with lots of great 
+        repository tooling. ✨)
+      `,
+			`
+      --owner (${chalk.cyanBright.bold(
+				"string",
+			)}): GitHub organization or user the repository is 
+        underneath (e.g. JoshuaKGoldberg)
+      `,
+			`
+      --repository (${chalk.cyanBright.bold(
+				"string",
+			)}): The kebab-case name of the repository 
+        (e.g. create-typescript-app)
+      `,
+			`
+      --title (${chalk.cyanBright.bold(
+				"string",
+			)}): Title Case title for the repository to be used in 
+        documentation (e.g. Create TypeScript App)
+      `,
+		),
+	);
+
+	logLine();
+
+	logLine(chalk.black.bgGreenBright("Optional options:"));
+
+	logLine(
+		chalk.cyan(
+			`
+      --author (${chalk.cyanBright.bold(
+				"string",
+			)}): Username on npm to publish packages under (by 
+        default, an existing npm author, or the currently logged in npm user, 
+        or ${chalk.cyanBright("owner.")}${chalk.magentaBright("toLowerCase()")})
```

Comment:
Out of curiosity, why `magentaBright` and for just the method call? I don't think we need to emphasize it so much.

(is it to match an editor theme?)

Reply:
Just because I used `cyanBright` to emphasize keywords like `string` and I thought to emphasize the snippet as code it should have two-color highlighting. But I'm not attached to any of the formatting if you have different preferences.

Reply:
Yeah I'd rather not emphasize them quite so much. It's quite a big jump in color.

Reply:
Ok! I've edited it so both parts of the code snippet are the same color.